content/docs/guides/components/tftraining.md

```diff
@@ -203,8 +204,34 @@ Typically you will want to change the following values
 
    * For example, you might need to configure various environment variables to talk to datastores like GCS or S3
 
-1. Attach PV's if you want to use PVs for storage.
+1. Attach PVs if you want to use PVs for storage.
 
+### Accessing the TFJob dashboard
+
+The TFJob dashboard is available at `<path>/tfjobs/ui/` (port `8080`).
+Specifically:
+
+* If you're using the central Kubeflow UI, you can access the TFJob dashboard
+  by clicking **TFJOB DASHBOARD**:
+
+    ![Central UI](/docs/images/central-ui.png)
+
+* If you followed the
+ [guide for Kubernetes Engine](/docs/started/getting-started-gke), you can
```

Comment:
I know we have "Kubernetes Engine for Kubeflow" in the other page, but I think we should just say GKE because "Kubernetes Engine" doesn't link to GKE in my head.

Also, the link is 404 for me in preview

Reply:
Good point - I've changed "Kubernetes Engine" to "GKE".

Hmm, I don't know what's happening with the links and image - they work for me. Is this the preview that you're using?
https://deploy-preview-300--competent-brattain-de2d6d.netlify.com/docs/guides/components/tftraining/

Reply:
weird, I was just clicking on the preview file button on github.
The above link works.

Reply:
Good! It works for me locally and in preview, so I think we're good.